tests/modules/test_Gatling.py

```diff
@@ -192,7 +192,12 @@ def test_requests_1(self):
                               }]},
                              {"url": "/",
                               "think-time": 2,
-                              "follow-redirects": True}]
+                              "follow-redirects": True},
+                             {"url": "/reserve.php",
+                              "method": "POST",
+                              "body": u"Body Content 2",
```

Comment:
In my point we have to check conversion with specific (non-ASCII) characters in unicode string.